src/data_collection.py

```diff
@@ -4,6 +4,24 @@
 Writes the data to a .json file.
 
 Calculates statistics based on the data from Github.
+
+Steps to run data_collection.py and display the data table:
+
+Must be in the repository folder.
+
+Run the python file using `pipenv run python src/data_collection.py`.
```

Comment:
It does not make sense to put the command in quotation marks since it is not a markdown file.

Reply:
I agree, however, these steps are just placeholders and they will be removed when the CLI team add their main function